pep-0649.rst

```diff
@@ -599,9 +601,9 @@ Language Reference:
     may raise ``NameError``; it must not raise ``NameError`` when called
     requesting any other format.
 
-    If an object doesn't have any annotations, ``__annotate__``
-    should preferably be deleted or set to ``None``, rather than set to
-    a function that returns an empty dict.
+    If an object doesn't have any annotations, ``__annotate__`` should
+    preferably be set to ``None``, rather than set to a function that returns an
+    empty dict.
```

Comment:
Does this read well if you add a parenthetical comment stipulating

(``__annotate__`` can't be deleted.)

Reply:
Done.